### ISSUE TITLE
Failure to build on Windows

### ISSUE THREAD

**Issue by lhsego** [tesseradata/install-vagrant#7]
After installing Vagrant (for Windows), Virtual Box, and cloning the repo per the instructions, `vagrant up` lead to a failure.  It never even got to the Tessera installation stuff.  Here's a screen shot

![unknown](https://cloud.githubusercontent.com/assets/5566573/5986453/13b0b018-a8ad-11e4-9aa5-0582674c9809.png)

**Comment by lhsego** [comment]
Lots of people have had this problem.  https://github.com/mitchellh/vagrant/issues/2157. Looks like a Virtual Box issue.  Wonder if VM ware would be a better choice...

**Comment by hafen** [comment]
Yes it looks like a vagrant / virtual box issue.  Do you have the latest versions of things? http://stackoverflow.com/questions/24793954/set-a-box-with-vagrant-virtualbox

VMWare costs money.  I don't want people to have to fork out money to run our virtual environment.  We could support it in addition to virtual box, but I don't know how much we have to change for that, and don't have time right now to investigate.  Have you tried some of the suggestions where other people have had the same problem?
